lib/Cleantalk/Common/HTTP/Request.php

```diff
@@ -88,7 +88,7 @@ public function setUrl($url)
     public function setData($data)
     {
         // If $data scalar converting it to array
-        $this->data = ! empty($data) && is_scalar($data)
+        $this->data = ! empty($data) && ! self::isJson($data) && is_scalar($data)
```

Comment:
Do we pass JSON string as data somewhere?

Reply:
Yes. Here https://github.com/CleanTalk/wordpress-antispam/pull/124/files#diff-4c5587519c7c7e39c2af11ee2c91b50f8ec5f6a44a2eedf4a9a713ce77bcfb4fR465